anagram/tests/anagram.rs

```diff
@@ -1,11 +1,6 @@
-#![crate_name = "anagram_test"]
-#![crate_type = "lib"]
-#![feature(collections)]
-
-mod anagram;
+extern crate anagram;
 
 #[test]
-#[ignore]
```

Comment:
We could put this as instructions in the SETUP.md file which gets included in each problem's README.md